user-org-es-util/src/main/java/org/sunbird/es/service/impl/ElasticSearchRestHighImpl.java

```diff
@@ -212,6 +212,7 @@ public void onResponse(GetResponse getResponse) {
                 promise.success(new HashMap<>());
```

Comment:
init hashmap with empty...and then have one return path. 3 promise.success and if-else conditions isn't good to read. Since there is no failure, we can init with empty.

Reply:
done